src/lib/xmlparse.rs

```diff
@@ -1394,9 +1644,9 @@ impl Drop for Tag {
    external protocol or NULL if there is none specified.
 */
 #[no_mangle]
-pub unsafe extern "C" fn XML_ParserCreate<'scf>(
+pub unsafe extern "C" fn XML_ParserCreate<'p>(
```

Comment:
Could we name this lifetime something a little more informative? I assume `'p` stands for parser? Maybe `'pars` or `'prsr`?

Reply:
Yep, I considered both `'p` and `'parser`. I don't mind the latter.